app/services/marketplace_service/available_currencies.rb

```diff
@@ -67,4 +67,6 @@ module MarketplaceService::AvailableCurrencies
       "VA" => "EUR"
   }
 
+  CURRENCIES = ["USD"].concat(COUNTRY_CURRENCIES.values).uniq.sort
```

Comment:
Feel that [SortedSet](https://ruby-doc.org/stdlib-1.9.3/libdoc/set/rdoc/SortedSet.html) could be the right data structure for this.